bundler/tool/bundler/dev_gems.rb.lock

```diff
@@ -3,6 +3,7 @@ GEM
   specs:
     diff-lcs (1.5.0)
     hpricot (0.8.6)
+    hpricot (0.8.6-java)
```

Comment:
Any idea why this was added? :thinking:

Reply:
No idea, but it seems correct, so maybe some bug fixed since this was last bundled?